pipelines/base/config-dump.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: dump-config
+spec:
+  params:
+  - name: REPO_NAME
+    type: string
+    default: .github
+  - name: BRANCH_NAME
+    type: string
+    default: add-initial-peribolos-config
+  - name: GIT_EMAIL
+    type: string
+  - name: GIT_NAME
+    type: string
+  workspaces:
+  - name: secret-data
+    description: location for installation secret
```

Comment:
Let's use `INSTALLATION_ID` param and mount the secret directly, like @mimotej did in #20 instead of a workspace

https://github.com/open-services-group/peribolos-as-a-service/pull/20/files#diff-094cd5f6b915fa506b26c95e471dc80c70e703c8bec8a78edf51732fdfcc9083R15-R25